DeepFilterNet/df/enhance.py

```diff
@@ -277,7 +286,7 @@ def setup_df_argument_parser(default_log_level: str = "INFO") -> argparse.Argume
         type=parse_epoch_type,
         help="Epoch for checkpoint loading. Can be one of ['best', 'latest', <int>].",
     )
-    parser.add_argument("--version", action="store_true")
+    parser.add_argument("--version", action=PrintVersion, nargs="?")
```

Comment:
Do we need `nargs` here?

Reply:
It's the simplest way to get it to work without producing the error `deepFilter: error: argument --version: expected one argument`. However, I've made an update to the `PrintVersion` class, so that `nargs` is not needed (I've also added a help string).

One thing I did just notice, is that using a custom class as an `action` seems to only have been introduced in Python 3.8, so if you want to maintain Python 3.7 compatibility this should wait until that is dropped.